common/data/defaultValues.ts

```diff
@@ -0,0 +1,18 @@
+// used for display purpose only, when the prismic data is undefined
+export const defaultArticleFormat = {
+  type: "ArticleFormat",
+  id: "W7TfJRAAAJ1D0eLK",
+  label: "Article",
+};
+
+export const defaultEventFormat = {
+  type: "EventFormat",
+  id: "dfc2b7f9-c362-47da-9644-b0f98212ccaa",
```

Comment:
If we're keeping this I'd be tempted to add a comment to explain the difference in IDs

Reply:
✅